docs/manage/partitions-data-tiers/infrequent-tier-slo-support.md

```diff
@@ -0,0 +1,47 @@
+---
+id: infrequent-tier-slo-support
+title: Infrequent Tier Support for SLOs
+description: Learn how to configure SLOs against the Infrequent data tier.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+Previously, SLOs were only supported in Sumo Logic's Continuous data tier. This support enabled you to configure SLOs on Infrequent tier logs to increase the monitoring coverage for reliability management.
+
+## Sample Queries
+
+While defining SLOs for the Infrequent tier logs, you can use the following sample queries.
+
+* Use `_dataTier=Infrequent` in your query scope. For example:
```

Comment:
1. Use `_dataTier=Infrequent` in your query scope. This ensures that your query only searches for data in the Infrequent tier. For example: